pkg/compute/build.go

```diff
@@ -52,6 +45,7 @@ func NewBuildCommand(parent common.Registerer, client api.HTTPClient, globals *c
 	c.CmdClause = parent.Command("build", "Build a Compute@Edge package locally")
 	c.CmdClause.Flag("name", "Package name").StringVar(&c.name)
 	c.CmdClause.Flag("language", "Language type").StringVar(&c.lang)
+	c.CmdClause.Flag("include-src", "Include src directory in package").BoolVar(&c.includeSrc)
```

Comment:
I think the name of this flag should be changed so that it's less specific to Rust. eg. If we changed the name to `--include-source`, we can generically apply it to other languages in the future. The description would need to be changed as well eg. `Include source code in built package`